xml/System.Composition.Convention/AttributedModelProvider.xml

```diff
@@ -12,8 +12,8 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Defines the core behavior of methods used to retrieve metadata attributes for a type, and provides a base for derived classes.</summary>
```

Comment:
This can be a bit clearer: Serves as the base class that defines the core behavior of methods used to retrieve metadata attributes for a type.